chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -348,16 +348,40 @@
 
 if !docker_compute_nodes.empty? && kvm_compute_nodes.empty?
   image_name = "cirros"
+  docker_nodes = docker_compute_nodes.map {|n| n.name}
 
-  bash "find tempest test image for docker" do
+  bash "load docker image" do
     code <<-EOH
-DOCKER_IMAGE_ID=$(glance #{insecure} image-list \
+
+TEMP=$(mktemp -d)
+IMG_FILE=$(basename $IMAGE_URL)
+
+echo "Downloading image ... "
+wget --no-verbose $IMAGE_URL --directory-prefix=$TEMP 2>&1 || exit $?
+
+echo "Deploying image in docker nodes ... "
+for node in $NODES ; do
+  scp $TEMP/$IMG_FILE $node:/tmp
```

Comment:
Unfortunatelly, this wasn't solved since last PR. tempest node doesn't have ssh access to the docker nodes....